assets/js/calypsoify.js

```diff
@@ -55,7 +55,27 @@
 	/**
 	 * Move page actions to action header.
 	 */
-	$( '.page-title-action, .add-new-h2' ).appendTo( '#action-header .action-header__actions' );
+	$( document ).ready( function() {
+		const $actions = $( '.page-title-action, .add-new-h2' );
+		if ( ! $actions.is( 'button' ) ) {
+			const buttons = [];
+			$actions.each( function( index, anAction ) {
+				const { className, href, text } = anAction;
```

Comment:
I _think_ there's no transpiling done in wc-calypso-bridge, so `const` will fail in IE11

Reply:
Nice catch! Thank you @becdetat! This was fixed in the commit 7b5ef73